docs/conversion/sakai_19_1-19_2_mysql_conversion.sql

```diff
@@ -1,3 +1,7 @@
 -- SAK-41878
 CREATE INDEX UK_f0kvf8pq0xapndnamvw5xr2ib ON ASN_SUBMISSION_SUBMITTER (SUBMITTER);
 -- SAK-41878
+
+-- SAK-41841
+ALTER TABLE RBC_CRITERION_OUTCOME MODIFY COMMENTS LONGTEXT;
```

Comment:
if this was created in lower-case, it should be referred to in lower-case:

CREATE TABLE rbc_criterion_outcome (